assets/style.css

```diff
@@ -0,0 +1,275 @@
+  margin: 8px;
+  padding: 8px;
+  color: white;
+  justify-content: space-evenly;
+  display: flex;
+}
+
+.navlink {
+  padding: 4px;
+  margin: 4px;
+  text-decoration: none;
+  color: white;
+}
+
+.navlink:hover {
+  color: lightblue;
+}
+
+.profile-pic {
+  width: 100px;
+  height: 100px;
+  border-radius: 10px 10px 10px 10px;
+  margin: 8px;
+}
+
+.latest-project {
+  background-image: url(./images/refactoring2.jpeg);
+  border: 2px solid black;
+  padding: 100px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  color: white;
+  border-radius: 10px 10px 10px 10px;
+}
+
+.latest-caption {
+  background-color: #5621f2;
+  font-size: 24px;
+  width: auto;
+  text-decoration-style: none;
+  color: white;
+}
+
+.latest-caption:hover {
+  color: red;
+}
+
+/* main content */
+
+.card-content {
+  display: flex;
+  margin: 20x;
+  padding: 8px;
+}
+
+.card {
+  justify-content: space-between;
+  margin: 10px;
+  padding: 10px;
+  width: 18rem;
+  background-color: #57585b;
+}
+
+.card:hover {
+  -webkit-transform: scale(1);
+  transform: scale(1);
+  -webkit-transition: 0.3s ease-in-out;
+  transition: 0.3s ease-in-out;
+  -webkit-transform: scale(1.3);
+  transform: scale(1.1);
+  background-image: var(--backgroundcolor);
+  box-shadow: 0 5px 15px rgba(145, 92, 182, 0.4);
+  color: white;
+}
+
+.card div .project-snapshot {
+  align-items: center;
+  --pic-width: 150px;
+  --pic-height: 100px;
+}
+
+.project-snapshot {
+  width: var(--pic-width);
+  height: var(--pic-height);
+}
+
+/* Footer & Socials */
+
+.footer-banner {
+  bottom: 0;
+  position: relative;
+  width: 100%;
+  justify-content: space-evenly;
+  color: white;
+  text-align: center;
+  background-image: var(--backgroundcolor);
+}
+
+.social-icons {
+  width: 50px;
+  margin: 12px;
+  opacity: 0.5;
+}
+
+.social-icons:hover {
+  transform: translateY(-5px);
+  opacity: 1;
+}
+
+/* Media Queries
+
+/* 	extra small resolutions (or xs) */
+@media screen and (max-width: 360px) {
+  .main-content,
+  .latest-project,
+  .card-content {
+    display: flex;
+    flex-direction: column;
+  }
+  .card {
+    padding-top: 24px;
+  }
+  .footer-banner {
+    position: relative;
+  }
+}
+
+/* 	small resolutions (or sm) */
+@media screen and (max-width: 768px) {
+  .main-content,
+  .latest-project,
+  .card-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+  .card {
+    padding-top: 24px;
+  }
+  .footer-banner {
+    position: relative;
+  }
+}
+
+/* 	medium resolutions (or md) */
+@media screen and (max-width: 1024px) {
+  .main-content,
+  .latest-project,
+  .card-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+  .card {
+    padding-top: 24px;
+  }
+  .footer-banner {
+    position: relative;
+  }
+}
+
+/* large resolutions (or lg) */
+@media screen and (max-width: 1280px) {
+  .main-content,
```

Comment:
Purely for style, it would be nice to see some sort of flex-wrap here, rather than just dropping the project cards to a column at 1280px, so that we can have 2 or 3 projects per row. Since your cards are quite small, they look quite lost on the page at this size. Another option would be to alter the size of the cards at this screen size, so they fill up more of the page width.